website/assets/js/copy.js

```diff
@@ -44,4 +52,18 @@ function addCopyButtonsToPres() {
   }
 }
 
+// getCommands splits the provided text into lines and returns the subset
+// of lines that represent commands, i.e., the lines that start with "$ ".
+function getCommands(text) {
+    let lines = text.split("\n");
+    let commands = Array()
+    for (let i = 0; i < lines.length; i++) {
+        let line = lines[i]
+        if (line.startsWith("$ ")) {
+            commands.push(line.substr(2))
```

Comment:
I think `substr` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr).  Replace with `substring`?